react/src/components/ui/buttons/Button/Button.module.css

```diff
@@ -1,5 +1,25 @@
+.button {
+  border: 1px solid;
+  border-radius: 4px;
+  font-family: arial, sans-serif;
+  font-size: 14px;
+  margin: 11px 4px;
+  padding: 0 16px;
+  line-height: 27px;
+  height: 36px;
+  min-width: 54px;
+  text-align: center;
+  cursor: pointer;
+  user-select: none;
+}
 
-  /* TODO: Component CSS Style goes here */
-  .line {
-    color: blue;
-  }
+.button.light {
+  background-color: #f8f9fa;
```

Comment:
add colors in the separate constants file

Reply:
@karthikeyanpa12 we can, because we haven't setup code to handle scss. I can do that change. Want to know what others think ?

CC: @ramsunvtech @yuvarajuv